### ISSUE TITLE
Language Translation

### ISSUE THREAD

**Issue by SonjaKoenig** [learnweb/moodle-mod_ratingallocate#19]
It's much easier to use AMOS for the translation process (to german). Therefore the plugin language packages !=en can be deleted.

**Comment by SonjaKoenig** [comment]
I tried to edit some of the english language pack at moodle -> mod -> ratingallocate -> lang -> en
However, the changes were not displayed until I copied the 'language file' into moodle -> lang -> en
Somehow though, there was no lanuage file there yet. So I was wondering anyways where the plugin got its english strings from if they were not from either of these two locations.
Any idea?

**Comment by tobiasreischmann** [comment]
The language pack at moodle -> mod -> ratingallocate -> lang -> en should be the one moodle uses. However, moodle uses a chaching mechanism, which prohibits any direct update, when changes to the language files are made. Are you aware of that? There are ways to circumvent this behaviour, e.g. by purging all cache as an admin or setting the value pair…
$CFG->langstringcache = false;
… in the config.php . The last thing should not be done on a production server!
The next possibility is, that there is another lang file under moodledata -> yourmoodle -> lang -> en.

Von: Sonja Koenig [mailto:notifications@github.com]
Gesendet: Montag, 23. März 2015 12:52
An: learnweb/moodle-mod_ratingallocate
Betreff: Re: [moodle-mod_ratingallocate] Translation (#19)


I tried to edit some of the english language pack at moodle -> mod -> ratingallocate -> lang -> en
However, the changes were not displayed until I copied the 'language file' into moodle -> lang -> en
Somehow though, there was no lanuage file there yet. So I was wondering anyways where the plugin got its english strings from if they were not from either of these two locations.
Any idea?
